tests/src/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -323,6 +324,27 @@ class WskEntitlementTests
             }
     }
 
+    it should "invoke a packaged action not owned by the subject to get the subject's namespace" in withAssetCleaner(guestWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) => pkg.create(samplePackage, shared = Some(true))(wp)
+            }
+
+            assetHelper.withCleaner(wsk.action, fullSampleActionName) {
+                val file = Some(TestUtils.getTestActionFilename("namespace.js"))
+                (action, _) => action.create(fullSampleActionName, file)(wp)
+            }
+
+            val fullyQualifiedActionName = s"/$guestNamespace/$fullSampleActionName"
+            val run = wsk.action.invoke(fullSampleActionName)(guestWskProps)
```

Comment:
I can't tell from this test alone (have to look at the values of `samplePackage` and `fullSampleActionName`
if the invoke is the desired one.

The test needs to invoke with key `A` an action in namespace `B`. The invoke here is using `guestWskProps` so let's call that key `A`. The action hence needs to be in namespace `B` - I don't think `fullSampleActionName` or `fullyQualifiedActionName` are correct in that case.